os/fs/driver/mtd/smart.c

```diff
@@ -493,73 +493,6 @@ static void smart_free(FAR struct smart_struct_s *dev, FAR void *ptr)
 }
 #endif
 
-/***************************************************************************************
- *   This is called from the fs_clean() API to release the memory acquired by MTD driver
- **************************************************************************************/
-int smart_clean(FAR struct inode *inode)
```

Comment:
fsAs far as I know, smart_clean() is not in use for now. So, Eunbong Song had decided to remove it. Please add it if you need it later.